src/paper/serializers.py

```diff
@@ -577,41 +535,30 @@ def update(self, instance, validated_data):
                 if file:
                     self._add_file(paper, file)
 
-                updated_hub_ids = list(
-                    map(lambda hub: hub.id, remove_hubs + new_hubs)
-                )
+                updated_hub_ids = list(map(lambda hub: hub.id, remove_hubs + new_hubs))
                 if len(updated_hub_ids) > 0:
                     reset_unified_document_cache(
                         hub_ids=updated_hub_ids,
-                        document_type=['paper', 'all'],
+                        document_type=["paper", "all"],
                         filters=[NEWEST, TOP, TRENDING, DISCUSSED],
+                        with_default_hub=True,
```

Comment:
I recommend you do not include this here in this case

Reply:
When the async flow finishes and the user has to input a hub, the paper won't show up in the feed without a feed cache reset. Is there something else that I should do instead?

Reply:
The reason behind this is that the homepage doesn't need to be updated that often. Given that it updates automatically every 30 mins does the trick in most cases..

There are times thought such as when a new paper is added though that we should update the HP feed cache immediately